storage/provider/common.go

```diff
@@ -12,9 +12,10 @@ import (
 // CommonProviders returns the storage providers used by all environments.
 func CommonProviders() map[storage.ProviderType]storage.Provider {
 	return map[storage.ProviderType]storage.Provider{
-		LoopProviderType:   &loopProvider{logAndExec},
-		RootfsProviderType: &rootfsProvider{logAndExec},
-		TmpfsProviderType:  &tmpfsProvider{logAndExec},
+		LoopProviderType:      &loopProvider{logAndExec},
+		RootfsProviderType:    &rootfsProvider{logAndExec},
+		TmpfsProviderType:     &tmpfsProvider{logAndExec},
+		OpenstackProviderType: &openstackProvider{newGooseAdapter},
```

Comment:
This doesn't belong here - common providers are those that work with any environment. The correct place to register this provider in the init() in providers/openstack. See how it's done for providers/ec2 and the ebs provider